src/Apps/Order/Components/ArtworkSummaryItem.tsx

```diff
@@ -81,14 +82,19 @@ const ArtworkSummaryItem: React.FC<ArtworkSummaryItemProps> = ({
             </Text>
           </>
         )}
-        {!isPrivateSale && artworkPrice && (
+        {!isPrivateSale && artworkPrice?.price && (
           <Text variant="sm">
             {`${priceLabel} ${appendCurrencySymbol(
               artworkPrice.price,
               currencyCode
             )}`}
           </Text>
         )}
+        {isPartnerOffer && !artworkPrice?.price && (
```

Comment:
How about not checking for partner offer here? That way we can have a nicer copy also in the case of traditional make offer on a non public price artwork, example:
<img width="694" alt="Screenshot 2023-12-08 at 15 37 19" src="https://github.com/artsy/force/assets/7518671/4e698263-63c3-47a4-aed3-b30778a94dd1">

```suggestion
        {!artworkPrice?.price && (
```